sklearn/utils/testing.py

```diff
@@ -73,6 +73,14 @@
 assert_less_equal = _dummy.assertLessEqual
 assert_greater_equal = _dummy.assertGreaterEqual
 
+
+for function_name in ('assert_equal', 'assert_not_equal', 'assert_greater',
+                      'assert_greater_equal', 'assert_less',
+                      'assert_less_equal', 'assert_in', 'assert_not_in'):
+    msg = ('{} is deprecated in version 0.22 ans will be removed in version '
+           '0.24. Please use "assert" instead'.format(function_name))
+    globals()[function_name] = deprecated(msg)(globals()[function_name])
```

Comment:
Manually modifying globals at import time seems not that ideal, 
```py
assert_equal = deprecated(msg)(_dummy.assertEquald)
```
is not that much longer and would be more explicit.

Reply:
I was trying to avoid having to change the custom message for each function. Would you be OK with just 

`msg = ('This helper is deprecated in version 0.22 ...`

The `@deprecated` decorator would prepend "blahblah is deprecated" to the message anyway.

Reply:
I think it should be OK these are mostly internal functions that shouldn't be used by users anyway.